public/app/features/alerting/unified/utils/receiver-form.ts

```diff
@@ -32,6 +37,40 @@ export function grafanaReceiverToFormValues(
   return [values, channelMap];
 }
 
+export function cloudReceiverToFormValues(
```

Comment:
nit: maybe give this function some <3 to make it easier to read.

Reply:
I added more comments for each step of the transform. Could not figure out how to break it down some more meaningfully though :/